telemetry-core/src/main/java/com/newrelic/telemetry/json/AttributesJson.java

```diff
@@ -16,7 +16,6 @@
   public String toJson(Map<String, Object> attributes) {
     StringWriter out = new StringWriter();
     Map<String, Object> filteredAttributes = filterIllegalValues(attributes);
-    filterIllegalValues(filteredAttributes);
```

Comment:
`filterIllegalValues` returns a new map with illegal values omitted. Called this here doesn't do anything.